designsafe/settings/common_settings.py

```diff
@@ -569,6 +569,12 @@ def guid_filter(record):
 TAPIS_ADMIN_JWT = os.environ.get('TAPIS_ADMIN_JWT')
 TAPIS_TG458981_JWT = os.environ.get('TAPIS_TG458981_JWT')
 
+# Core Portal Configuration
+CORE_PORTAL_API_BASE = os.environ.get("CORE_PORTAL_API_BASE", "https://cep.tacc.utexas.edu/api/system-monitor")
```

Comment:
If you take a look at [how this API is used in the Core Portal](https://github.com/TACC/Core-Portal/blob/a661a0174c921e844fdff84f1c3106b4d4c1c5bc/server/portal/apps/system_monitor/views.py#L28), you'll see that it uses the [same TAP URL ](SYSTEM_MONITOR_URL)that you're setting below. Is there a reason to use this URL, or can you get the same information from the TAP API? 

It's good to use a single source of "truth" for this information, instead of passing it from portal to portal.